azure-pipelines.perf-test.yml

```diff
@@ -0,0 +1,62 @@
+pr: none
+
+trigger: none
+
+variables:
+  - group: fabric-variables
+
+pool:
+  vmImage: 'Ubuntu 18.04'
+
+steps:
+  # No need for checkout since we're using build artifacts.
+  - checkout: none
+
+  - template: azure-pipelines.artifacts.yml
+    parameters:
+      artifact: $(artifact)
+      buildId: $(buildId)
+
+  - template: azure-pipelines.tools.yml
+
+  - script: |
+      yarn
+    displayName: yarn
```

Comment:
I thought I took this out in #11574. If this pipeline is downloading artifacts, it shouldn't need to run `yarn`. None of the other jobs do.

Reply:
I'll take it out and then merge if green.